test/e2e/fetch-cache-limit/fetch-cache-limit.test.ts

```diff
@@ -0,0 +1,22 @@
+import { createNextDescribe } from 'e2e-utils'
```

Comment:
We could add a check for this to the existing [test suite here](https://github.com/vercel/next.js/blob/canary/test/e2e/app-dir/app-static/app-static-custom-handler.test.ts)

Reply:
Thanks, done